src/knockout/components/action-bar/action-bar.ts

```diff
@@ -26,25 +34,32 @@ export class ActionBarViewModel extends ActionBar {
 }
 
 export class AdaptiveElementImplementor extends ImplementorBase {
-  constructor(model: AdaptiveElement) {
+  constructor(model: AdaptiveElement | AdaptiveActionContainer) {
     super(model);
 
-    model.items.forEach((item) => {
-      new ImplementorBase(item.stateItem);
+    ((<any>model).items || (<any>model).actions).forEach((item: any) => {
+      if (!!item.stateItem) {
+        new ImplementorBase(item.stateItem);
+      } else {
+        new ImplementorBase(item);
+      }
```

Comment:
We need to wrap it with computed I think, otherwise we'll not update if actions will be modified